tests/manager/feed/test_annotators.py

```diff
@@ -409,6 +410,32 @@ def test_annotate_work_entry(self, db: DatabaseTransactionFixture):
         edition: Edition = work.presentation_edition
         now = utc_now()
 
+        def create_sample_link(media_type: str, url: str):
+            representation, _ = get_one_or_create(
+                db.session, Representation, url=url, media_type=media_type
+            )
+            resource, _ = get_one_or_create(
+                db.session,
+                Resource,
+                url=url + "/preview",
+                data_source=edition.data_source,
+                representation=representation,
+            )
+            link = get_one_or_create(
+                db.session,
+                Hyperlink,
+                identifier=edition.primary_identifier,
+                data_source=edition.data_source,
+                rel=Hyperlink.SAMPLE,
+                resource=resource,
+            )
+            return link
+
+        create_sample_link(MediaTypes.PDF_MEDIA_TYPE, "http://pdf")
+        create_sample_link(MediaTypes.EPUB_MEDIA_TYPE, "http://epub")
+        create_sample_link(MediaTypes.TEXT_HTML_MEDIA_TYPE, "http://html")
+        create_sample_link(MediaTypes.APPLICATION_JSON_MEDIA_TYPE, "http://json")
```

Comment:
It might be useful to include multiple `epub` and multiple of some other type to completely test the comparitor. In that case, the assertions below might need to check the `href`s, as well, to ensure that we get the order we're expecting.

Reply:
Revising this comment, based on my recent comment about not needing to do anything except sort the epubs to the top. I don't think we need to have more than one of any type except epub, so I think this one is resolved.